Extension/src/LanguageServer/client.ts

```diff
@@ -1294,20 +1299,34 @@ export class DefaultClient implements Client {
         this.rootFolder = workspaceFolder;
         this.rootRealPath = this.RootPath ? (fs.existsSync(this.RootPath) ? fs.realpathSync(this.RootPath) : this.RootPath) : "";
 
-        let storagePath: string | undefined;
+        const baseStoragePath: string | undefined = util.getBaseStoragePath();
+        let legacyStoragePath: string | undefined;
```

Comment:
This doesn't need to be undefined if it's initialized to "" like the storagePath is (but see my other comment about potentially removing the variable).